src/components/FlowChart/FlowSelection.tsx

```diff
@@ -112,47 +104,39 @@ export default function FlowSelection({
   const currentFlow = flows[selectedFlowId];
 
   return (
-    <Dropdown className="flow-selection">
-      <Dropdown.Toggle>
+    <FlowSelectionMenu>
+      <SelectedFlowHeader>
         {currentFlow.name}
         <TitleConnectionIndicator
           show={currentFlow.type === "remote"}
           connected={connected}
         />
-      </Dropdown.Toggle>
-      <Dropdown.Menu>
-        <Dropdown.Item onClick={createNewFlow}>
-          <i className="material-icons">add</i>New Flow
-        </Dropdown.Item>
-        <Dropdown.Header className="text-uppercase">
-          Your Flows{" "}
-          <EditFlowsButton
-            isEditing={isEditing}
-            toggleEditing={toggleEditing}
+      </SelectedFlowHeader>
+
+      <FlowTap onClick={showNewFlowModal}>
+        New Flow <i className="material-icons plus-icon">add</i>
+      </FlowTap>
+
+      <FlowHeader>My flow </FlowHeader>
+
+      {userFlows.map(([flowId, flow], idx) => (
+        <FlowTap onClick={() => loadFlow(flowId)} key={idx}>
+          {flow.name}
```

Comment:
Would be good to see the currently selected flow name in bold or something

Reply:
done